docs/features/import_export.md

```diff
@@ -1,9 +1,9 @@
-This application features a very versatile import and export feature in order 
+This application features a very versatile import and export feature in order
 to offer the best experience possible and allow you to freely choose where your data goes.
 
 !!! WARNING "WIP"
-    The Module is relatively new. There is a known issue with [Timeouts](https://github.com/vabene1111/recipes/issues/417) on large exports.
-    A fix is being developed and will likely be released with the next version.
+The Module is relatively new. There is a known issue with [Timeouts](https://github.com/vabene1111/recipes/issues/417) on large exports.
```

Comment:
please discard these changes and all others that strip spaces in !!!! warning/.. blocks as it will break the formatting